client/maisie/resources/users.py

```diff
@@ -1,4 +1,11 @@
+import json
+import logging
+
+from typing import Union
 from maisie import BaseAction
+from maisie.utils.git import GitProvider
```

Comment:
Why do we need to import `GitProvider`? I believe all operations are already handled by `BaseAction`, aside from maybe the `Models` action class in `models.py`, where additional functionality is required, thus the import is present.